===================================================================
RCS file: /home/kde/kdepim/Makefile.am.in,v
retrieving revision 1.11
retrieving revision 1.12
diff -u -r1.11 -r1.12
--- kdepim/Makefile.am.in	2002/08/08 09:40:48	1.11
+++ kdepim/Makefile.am.in	2002/09/29 07:53:00	1.12
@@ -11,6 +11,7 @@
 COMPILE_BEFORE_libkcal = libical
 COMPILE_BEFORE_kaplan = kaddressbook knotes korganizer
 COMPILE_BEFORE_libkpimexchange = kalarmd
+COMPILE_BEFORE_kalarm = kalarmd
 
 AUTOMAKE_OPTIONS = foreign 1.5
 DISTCLEANFILES = inst-apps
